### PR TITLE
Coalesce Incremental Load Fields in `shopify__customer_cohorts`

### PR DESCRIPTION
**Pull request by advolut-team**
**Please provide your name and company**

**Link the issue/feature request which this PR is meant to address**
[Issue](https://github.com/fivetran/dbt_shopify/issues/85)

**Detail what changes this PR introduces and how this addresses the issue/feature request linked above.**
Coalesces the previous lifetimes fields from incremental loads. This fixes the issue of NULL values in the lifetime columns in `shopify__customer_cohorts` table.

**How did you validate the changes introduced within this PR?**
Ran `dbt run` locally and I no longer see NULLs in my Fivetran connector warehouse. 

**Which warehouse did you use to develop these changes?**
AWS Redshift

**Did you update the CHANGELOG?**

- [ ] Yes

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)**

- [ ] Yes

**Provide an emoji that best describes your current mood**
 
🏃 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

**PR Template** 
- [Community Pull Request Template](?expand=1&template=pull_request_template.md) (default)

- [Maintainer Pull Request Template](?expand=1&template=maintainer_pull_request_template.md) (to be used by maintainers)

### CODE REVIEW

**Comment by fivetran-reneeli** [comment]
Thanks @advolut-team for opening this! Our team will work on this in our upcoming sprint!

File: models/shopify__customer_cohorts.sql [hunk 1]
```diff
@@ -103,10 +103,10 @@ with calendar as (
         windows.order_count_in_month,
         windows.total_price_in_month,
         windows.line_item_count_in_month,
-        backfill_lifetime_sums.previous_cohort_month_number + windows.cohort_month_number as cohort_month_number,
-        backfill_lifetime_sums.previous_total_price_lifetime + windows.total_price_lifetime as total_price_lifetime,
-        backfill_lifetime_sums.previous_order_count_lifetime + windows.order_count_lifetime as order_count_lifetime,
-        backfill_lifetime_sums.previous_line_item_count_lifetime + windows.line_item_count_lifetime as line_item_count_lifetime,
+        coalesce(backfill_lifetime_sums.previous_cohort_month_number, 0) + windows.cohort_month_number as cohort_month_number,
```
Comment (fivetran-avinash) [inline models/shopify__customer_cohorts.sql:106]: Do you think we should coalesce ` windows.cohort_month_number` in the rarer (but possible) case that this value returns null on an incremental run? There could be an incremental period where no orders are brought in during the window and thus the partition is run on an empty table.

**Review by fivetran-avinash — CHANGES_REQUESTED** [review]
Hi @advolut-team , thanks for submitting this PR! I have one question before moving forward with approval and merging it into a release branch for our internal processes. 

Would you also be able to do the following:
- Bump the version in both `dbt_project.yml` and `integration_tests/dbt_project.yml` to 0.13.1
- Add the changes you made in the CHANGELOG. You can use the "Detail what changes this PR introduces" section of the PR and elaborate if you'd like. 

Let me know if you have any questions.

File: models/shopify__customer_cohorts.sql (hunk 1 above)
Comment (advolut-team) [reply models/shopify__customer_cohorts.sql:106]: Definitely a rare case, but I think there's no harm adding a `coalesce` statement to maintain data quality :) I've tested locally by running `dbt run` with `coalesce(windows. cohort_month_number)` and I have the same result. Let me add that in my next commit, along with the CHANGELOG and version bump

**Review by fivetran-avinash — APPROVED** [review]
Awesome, thanks for replying to all my comments @advolut-team ! I'll go ahead and merge these changes into a development branch and will hopefully have this live in a new release on the dbt hub soon.

**Pull request merged**